<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventanaInspectores</class>
 <widget class="QMainWindow" name="ventanaInspectores">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>205</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>378</width>
    <height>205</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>378</width>
    <height>205</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Inspectores</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imagenes.qrc">
    <normaloff>:/imagenes/Imagenes/iconoInspector.png</normaloff>:/imagenes/Imagenes/iconoInspector.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="Probabilidad">
            <property name="text">
             <string>Probabilidad</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="aprobadas">
            <property name="text">
             <string>Aprobadas</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="Desechadas">
            <property name="text">
             <string>Desechadas</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLineEdit" name="editorProbabilidad">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="aprobadasValor">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="desechadasValor">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="radioButton">
          <property name="text">
           <string>Encendido</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_2">
          <property name="text">
           <string>Apagado</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="configurar">
          <property name="text">
           <string>Configurar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Aceptar">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Aceptar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
